package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "strapi-blurhash",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
I'm usure about how you handle versions, I just guessed `1.2.0`. Tell me if it's wrong.

Reply:
And also remember to deploy the new version to npm after merging